docs/development/developer-guide.rst

```diff
@@ -26,10 +26,10 @@ Python Packaging Authority (PyPA) with several core contributors. All bugs
 for Setuptools are filed and the canonical source is maintained in GitHub.
 
 User support and discussions are done through the issue tracker (for specific)
-issues, through the `distutils-sig mailing list <https://mail.python.org/mailman3/lists/distutils-sig.python.org/>`_, or on IRC (Freenode) at
-#pypa.
+issues, through `GitHub Discussions <https://github.com/pypa/setuptools/discussions>`_,
+or on IRC (Freenode) at #pypa.
 
-Discussions about development happen on the distutils-sig mailing list or on
+Discussions about development happen on GitHub Discussions or
 `Gitter <https://gitter.im/pypa/setuptools>`_.
```

Comment:
Could I ask you to take this opportunity and also change the chat URL to? 😝 

```suggestion
`Gitter <https://gitter.im/pypa/setuptools>`_.
the ``setuptools`` channel on `PyPA Discord <https://discord.com/invite/pypa>`_.
```

Reply:
Sure, is the preferred way that I just use the `commit suggestion` under this suggested change?

Reply:
You can use the `commit suggestion`, I just left it open to you in the case you wanted to change it in a different way.

Reply:
Pushed a slightly different version that flows better.

Reply:
Sorry, my mistake when I was suggesting the change I forgot to remove the line with the `gitter` (supposed to be replaced with discord).

Communication via `gitter` has been deprecated. Additionally, I also noticed that Freenode seems to be down, so I submitted a commit removing those 2.

@ashemedai, please let me know if you agree with the changes I have pushed.